10.9-libcxx/stable/main/finkinfo/languages/gcc12-13.0.info

```diff
@@ -0,0 +1,315 @@
+Info2: <<
+Package: gcc12
+Version: 12.3.0
+Revision: 1
+Type: gccver (12)
+Distribution: 13.0
+Source: mirror:gnu:gcc/gcc-%v/gcc-%v.tar.xz
+Source-Checksum: SHA256(949a5d4f99e786421a93b532b22ffab5578de7321369975b91aec97adfda8c3b)
+PatchFile: %n.patch
+PatchFile-MD5: 7d533489637df18650d997f3139c5772
```

Comment:
Incorrect MD5 for patch - should be e245cee9fa6aa2fd5298d7712358e8e1

Reply:
Thanks, fixed!

Reply:
PatchFile-MD5's still don't seem to match in gcc12-13.0.info, and between gcc13.info, gcc13-13.0.info, and gcc13-14.0.info